cardano-crypto-praos/src/Cardano/Crypto/VRF/Praos.hs

```diff
@@ -429,12 +429,12 @@ instance VRFAlgorithm PraosVRF where
 
   deriveVerKeyVRF = coerce skToVerKey
 
-  evalVRF = \_ msg (SignKeyPraosVRF sk) -> do
+  evalVRF = \_ msg (SignKeyPraosVRF sk) ->
     let msgBS = serialize' msg
-    proof <- maybe (error "Invalid Key") pure $ prove sk msgBS
-    output <- maybe (error "Invalid Proof") pure $ outputFromProof proof
-    return $ output `seq` proof `seq`
-             (OutputVRF (outputBytes output), CertPraosVRF proof)
+        proof = maybe (error "Invalid Key") id $ prove sk msgBS
+        output = maybe (error "Invalid Proof") id $ outputFromProof proof
+    in output `seq` proof `seq`
```

Comment:
Unrelated to this PR, but we should prefer bang patterns on the let bound vars to the use of seq.